src/QtAVPlayer/qavhwdevice_vaapi_drm_egl.cpp

```diff
@@ -16,7 +16,12 @@
 #include <GLES2/gl2.h>
 #include <GLES2/gl2ext.h>
 #include <va/va_drmcommon.h>
+
+#ifdef USE_LIBDRM
+#include <libdrm/drm_fourcc.h>
+#else
 #include <drm/drm_fourcc.h>
```

Comment:
```bash
val@qt:~$ dpkg -S /usr/include/drm/drm_fourcc.h
linux-libc-dev:amd64: /usr/include/drm/drm_fourcc.h
val@qt:~$ dpkg -S /usr/include/libdrm/drm_fourcc.h
libdrm-dev:amd64: /usr/include/libdrm/drm_fourcc.h
```
Looks it is too different packages, so I would suggest to use
`#include <drm_fourcc.h>`
but in cmake/pro file to add include path to found package, instead of inventing the define.

How you like this idea?

Reply:
Acho que eu entendi. Nós temos:
drm/drm_fourcc.h
libdrm/drm_fourcc.h

você quer que o compilador escolha entre libdrm ou drm no arquivo cmake/pro.
mas aí teria que mudar para #include <drm_fourcc.h>

Reply:
Yes only this should be used,
> #include <drm_fourcc.h>

You can try to make it for cmake, will try to think how to adjust qmake pro.

Reply:
ok